### ISSUE TITLE
.KSS file script

### ISSUE THREAD

**Issue by K4KDR** [f4gkr/PicTalk#20]
No doubt there are better ways to do this, but if anyone would like to convert all or part of the **pictalkframes.txt** data output file created by PicTalk to a **.KSS** file for use in other apps (i.e., DK3WN decoder, etc.), please feel free to use (and IMPROVE!) a simple script I uploaded to:  https://git.io/vAKfj

**Comment by f4gkr** [comment]
Thanks for the proposal.
Can you point me some specs of the KSS file format ?

Thanks

2018-02-26 18:06 GMT+01:00 K4KDR <notifications@github.com>:

**Comment by K4KDR** [comment]
Absolutely!  Please see the KISS section at:  http://destevez.net/2016/06/kiss-hdlc-ax-25-and-friends/

**Comment by f4gkr** [comment]
Thanks for the link
